pkg/template/event.go

```diff
@@ -169,26 +169,65 @@ func getHeaderValue(header map[string][]string, headerName string) (string, erro
 func ResolveParams(bindings []*triggersv1.TriggerBinding, body []byte, header map[string][]string, params []pipelinev1.ParamSpec) ([]pipelinev1.Param, error) {
 	out, err := MergeBindingParams(bindings)
 	if err != nil {
-		return nil, xerrors.Errorf("error merging trigger params: %v", err)
+		return nil, xerrors.Errorf("error merging trigger params: %q", err)
 	}
-	out, err = ApplyBodyToParams(body, out)
```

Comment:
Only used in tests. Wanted to remove it in a different PR to keep the size of this small but can also do it here.

Reply:
I think it's fine to remove it here. Simple deletions are easy to grok, and I'd rather have a larger PR than someone accidentally reading / trying to modify a dead piece of code. Also helps in the event of a rollback (only 1 commit to revert instead of 2).